pkg/generic/generic.go

```diff
@@ -118,11 +122,21 @@ func SetBinaryWithBase64(g Generic, enable bool) error {
 			return fmt.Errorf("empty codec for %#v", c)
 		}
 		c.codec.binaryWithBase64 = enable
+		if c.codec.dynamicgoEnabled {
```

Comment:
now we have 2 `dynamicgoEnabled` field? one in codec and the other in provider? a bit confusing

Reply:
? just passing the bool from the upper level to the lower level. (provider option -> codec -> read/write)

Reply:
Yeah, after read the whole PR I think it's OK